pkg/multiclustermanager/capabilities/http_request_body.go

```diff
@@ -5,8 +5,29 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/sirupsen/logrus"
 )
 
+type errorResponse struct {
+	Error string `json:"error"`
+}
+
+func handleErr(writer http.ResponseWriter, originalErr error) {
+	resp := errorResponse{originalErr.Error()}
+
+	asJSON, err := json.Marshal(resp)
+
+	if err != nil {
+		logrus.Error("error while marshalling error message '" + originalErr.Error() + "' error was '" + err.Error() + "'")
+		writer.Write([]byte(err.Error()))
+
+		return
+	}
+
+	writer.Write([]byte(asJSON))
```

Comment:
`json.Marshal` returns `[]byte`.
```suggestion
	writer.Write(asJSON)
```